news/2023/2023-05-13-new-featured-artist-cygnus.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "New Featured Artist: cygnus"
+date: 2023-05-13 12:00:00 +0000
+---
+
+Do you believe in simulation theory? If not, disregard this message and carry on with your day. Otherwise, you might want to check out our newest Featured Artist.
+
+![](https://assets.ppy.sh/artists/359/header.jpg)
+
+**cygnus** (known here on osu! as [Matrix](https://osu.ppy.sh/users/5052899)) is the embodiment of the circle clicker's dream: living in Japan and producing rhythm game songs all day every day.
+
+While he began as a pioneer of AR11 gameplay and later shifted his attention towards mapping, he's recently discovered that music is his favourite form of expression. Inspired by creators like our very own [*-45*](https://osu.ppy.sh/beatmaps/artists/297), *onoken*, and *SHIKI*, **cygnus** pulls off a distinctly old-school style through both his compositions and nostalgic sample choices.
+
+**23** tracks are patiently waiting for your editor magic on [**cygnus**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/359)!
+
+Given **cygnus**'s presence in the mapping community, it's no surprise that he wrangled a fair share of the mapping community into creating maps for his music. Below are a few examples of the awesome maps that have been made in the last few months, including maps by the artist himself:
+
+<div align="center">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/artists/359/release_showcase.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+### cygnus - STRIKER+
+
+Play through [this soon-to-be-ranked map](https://osu.ppy.sh/beatmapsets/1966073) with an 8-star top difficulty and guest difficulties by the no-caps-crew: [rrtyui](https://osu.ppy.sh/users/352328), [mithew](https://osu.ppy.sh/users/2571893), [semaphore](https://osu.ppy.sh/users/6313643), and [fanzhen0019](https://osu.ppy.sh/users/418699)!
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/359/Evolution%20of%20War%2Fcygnus%20-%20STRIKER%2B.mp3">
+</audio>
+
+### cygnus - The Evolution of War
+
+Resident 4K Featured Artist mapper [FAMoss](https://osu.ppy.sh/users/7707789) is here to deliver [this map](https://osu.ppy.sh/beatmapsets/1987180) to all of osu!'s piano smashers.
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/359/Evolution%20of%20War%2Fcygnus%20-%20The%20Evolution%20of%20War.mp3">
```

Comment:
what's up with the backslashes being encoded as `%2F`? https://assets.ppy.sh/artists/359/Evolution%20of%20War/cygnus%20-%20The%20Evolution%20of%20War.mp3 is equivalent and is how it has been done before

from a cursory look over fa posts during last year this seems to have been going on for a while, but it seems to be a bit random for some reason, with both ways appearing in the same article such as [this one](https://github.com/ppy/osu-wiki/blob/master/news/2022/2022-07-27-new-featured-artist-pegboard-nerds.md?plain=1#L46-L54)

i may do a pr to clean these up at some point, but it's fairly inconsequential

Reply:
sometimes i overshoot when using vscode's urlencode plugin, that's all it is

fairly inconsequential either way i think, though `%2F` should probably always be `/` just for readability's sake